src/main/java/org/jenkinsci/maven/plugins/hpi/HplMojo.java

```diff
@@ -111,39 +111,39 @@ public void execute() throws MojoExecutionException, MojoFailureException {
-            if (artifact.isPluginBestEffort(getLog()))
-                jenkinsPlugins.add(artifact.getId());
-        }
-
-        OUTER:
-        for (MavenArtifact artifact : artifacts) {
-            if(jenkinsPlugins.contains(artifact.getId()))
-                continue;   // plugin dependencies
-            if (artifact.getDependencyTrail().size() < 2) {
-                throw new IllegalStateException(
-                        "invalid dependency trail: " + artifact.getDependencyTrail());
-            }
-            if(artifact.getDependencyTrail().size() >= 1 && jenkinsPlugins.contains(artifact.getDependencyTrail().get(1)))
-                continue;   // no need to have transitive dependencies through plugins
-
-            // if the dependency goes through jenkins core, that's not a library
-            for (String trail : artifact.getDependencyTrail()) {
-                if (trail.contains(":hudson-core:") || trail.contains(":jenkins-core:"))
-                    continue OUTER;
-            }
-
-            ScopeArtifactFilter filter = new ScopeArtifactFilter(Artifact.SCOPE_RUNTIME);
-            if (!artifact.isOptional() && filter.include(artifact.artifact)) {
-                paths.add(artifact.getFile().getPath());
-            }
-        }
-    }
+    // private void buildLibraries(List<String> paths) throws IOException {
+    //     Set<MavenArtifact> artifacts = getProjectArtfacts();
+
+    //     // List up IDs of Jenkins plugin dependencies
+    //     Set<String> jenkinsPlugins = new HashSet<>();
+    //     for (MavenArtifact artifact : artifacts) {
+    //         if (artifact.isPluginBestEffort(getLog()))
+    //             jenkinsPlugins.add(artifact.getId());
+    //     }
+
+    //     OUTER:
+    //     for (MavenArtifact artifact : artifacts) {
+    //         if(jenkinsPlugins.contains(artifact.getId()))
+    //             continue;   // plugin dependencies
+    //         if (artifact.getDependencyTrail().size() < 2) {
+    //             throw new IllegalStateException(
+    //                     "invalid dependency trail: " + artifact.getDependencyTrail());
+    //         }
+    //         if(artifact.getDependencyTrail().size() >= 1 && jenkinsPlugins.contains(artifact.getDependencyTrail().get(1)))
+    //             continue;   // no need to have transitive dependencies through plugins
+
+    //         // if the dependency goes through jenkins core, that's not a library
+    //         for (String trail : artifact.getDependencyTrail()) {
+    //             if (trail.contains(":hudson-core:") || trail.contains(":jenkins-core:"))
+    //                 continue OUTER;
+    //         }
+
+    //         ScopeArtifactFilter filter = new ScopeArtifactFilter(Artifact.SCOPE_RUNTIME);
+    //         if (!artifact.isOptional() && filter.include(artifact.artifact)) {
+    //             paths.add(artifact.getFile().getPath());
+    //         }
+    //     }
+    // }
```

Comment:
Since the stated goal of the pull request is to improve readability, shouldn't the large comment block be removed?
```suggestion
```